src/spec/payid-protocol.md

```diff
@@ -156,12 +156,12 @@ This specification defines the PayID protocol - an application-layer protocol, w
        required addressDetailsType addressDetails
       }
 
-   * paymentNetwork: The value of payment-network as specified in the client request's `Accept` header
-   * environment: The value of environment as specified in the client request's `Accept` header
+   * paymentNetwork: The value of `paymentNetwork` as specified in the client request's `Accept` header
```

Comment:
We canonically refer to the payment network as `paymentNetwork` in the reference server and the documentation website.